javascript/javascript1/week2/get-students.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+function addStudentToClass(studentName){
+    if(class07Students.length ===6 && studentName!=="Queen")
+       { return console.log(`Can not  add  more students to class 07`);
+    }
+    else if(class07Students.includes(studentName)){
+       return console.log(`Student ${studentName} is already in the class`); 
+    }
+    else if(studentName==="Queen"){
+    class07Students.push(studentName)
+   return console.log(`${studentName} can be added to class 07`);
+}
+    
+
+    else if(studentName===" "){
+        return console.log(`You can not add empty string to the calss`)
+    }
+    else{
+     console.log(class07Students.push(studentName));
```

Comment:
There is some indenting that has gone wrong on these lines making the function quite hard to read.